document/CoreFormat.mdk

```diff
@@ -361,12 +361,70 @@ A `result` object describes a single result produced by a static analysis tool.
 ### `ruleId` property { #result-ruleId }
 
 A `result` object shall contain a property named `ruleId` whose value is a string containing
-a stable, opaque identifier for the rule that was evaluated to produce the result.
+the stable, opaque identifier for the rule that was evaluated to produce the result.
 
 ~ IsoExample
 `"CA2101"`
 ~
 
+### `kind` property { #result-kind }
+
+A `result` object may contain a property named `kind` whose value is one of a fixed set of strings
+that specify the category of the result.
+
+If present, the `kind` property shall have one of the following values, with the specified meanings:
+
+* `pass`: The rule specified by the `ruleId` property (&sect;[#result-ruleId]) was evaluated, and no problem was found.
+
+* `warning`: The rule specified by the `ruleId` property was evaluated, and a problem was found.
+
+* `error`: The rule specified by the `ruleId` property was evaluated, and a serious problem was found.
+
+* `pending`: The rule specified by the `ruleId` property was evaluated, but whether the result of the evaluation
+constitutes a problem cannot be determined without further evaluation, either by a person or by an automated tool.
+
+~ IsoNote
+After further evaluation, a person or an automated tool might change the `kind` property from `pending` to
+`warning`, `error`, or `pass`.
+~
+
+~ IsoNote
+In some cases, team policy or corporate policy might determine if an analysis result constitutes a problem.
+~
+
+~ IsoExample
+A binary checker might produce a result stating that an analysis target was compiled with a particular version of the compiler.
+Subsequently, a post-processing tool might examine this result to determine whether the use of that version of the compiler
+complies with corporate policy.
+~
+
+* `notApplicable`: The rule specified by the `ruleId` property was not evaluated, because it does not apply to the
```

Comment:
You might provide an example here. For example, a security check might apply to 32-bit binaries only. Such a tool would produce a 'notApplicable' message when passed a 64-bit binary to inspect.